p/themes/Origine/origine.css

```diff
@@ -184,7 +183,6 @@ form th {
 
 a.btn {
 	min-height: 25px;
```

Comment:
Presumably not necessary?

Reply:
it is copied from base-theme/base.css

Reply:
In that case it's better to either leave the entire definition for another time or to fix both at once.